File: Daily Challanges/2026-05-May/2026-05-05.py
def is_narcissistic(n):
    digits = list(map(int, str(n)))
    n_digits = len(digits)
    raised_sum = 0
    for digit in digits:
        raised_sum += digit ** n_digits
    return raised_sum == n

print(is_narcissistic(153))
print(is_narcissistic(154))

File: Daily Challanges/2026-05-May/2026-05-05.js
function isNarcissistic(n) {
  const digits = String(n)
    .split("")
    .map((item) => Number(item));
  const n_digits = digits.length;
  const raised_sum = digits.reduce((sum, digit) => sum + digit ** n_digits, 0);
  console.log(raised_sum);
  return raised_sum == n;
}

console.log(isNarcissistic(153));
console.log(isNarcissistic(154));
